pkg/daemon/ceph/client/pool.go

```diff
@@ -283,6 +292,14 @@ func CreateReplicatedPoolForApp(context *clusterd.Context, namespace, poolName s
 		}
 	}
 
+	compressionModeProperty := "compression_mode"
+	if pool.CompressionMode != "" {
+		err = SetPoolProperty(context, namespace, poolName, compressionModeProperty, pool.CompressionMode)
```

Comment:
Does compression work for both nautilus and octopus? Or what is the min version?

Reply:
Yes, I think the minimum version is mimic (although luminous docs. also carry the settings).